### PR TITLE
Add Pella to Master

### LINKED ISSUE #46: Readiu - Pella, IA
## Pella, IA 

- [ ] Visit Pella
- [ ] Visit Central College
- [ ] Visit the Windmill
- [ ] Go home
- [ ] Forget you ever came!

### PR DESCRIPTION
**Pull request by readiu**
Closes #46

### CODE REVIEW

**Comment by readiu** [comment]
@jeremiahmegel Hey You!!!

**Comment by jeremiahmegel** [comment]
@readiu Who are you???

File: places/Readiu-Pella.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Pella
+
+### Place to see
+- Windmill
+- Phonebooth
+- Redrock
+### Places to go
+-Central College
+### What to do
+-Jump in a lake
```
Comment (tony-lein) [inline places/Readiu-Pella.md]: Sounds nice

File: places/Readiu-Pella.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## Pella
+
+### Place to see
+- Windmill
+- Phonebooth
```
Comment (jeremiahmegel) [inline places/Readiu-Pella.md]: Disclaimer: no phone inside

**Comment by jeremiahmegel** [comment]
Go ahead and :shipit:!

**Pull request merged**